backend/src/test/java/ch/puzzle/okr/controller/ObjectiveControllerIT.java

```diff
@@ -44,13 +52,21 @@ class ObjectiveControllerIT {
     private static final String URL_BASE_OBJECTIVE = "/api/v2/objectives";
     private static final String URL_OBJECTIVE_5 = "/api/v2/objectives/5";
     private static final String URL_OBJECTIVE_10 = "/api/v2/objectives/10";
+    private static final String URL_DUPLICATE_OBJECTIVE_5 = "/api/v2/objectives/5";
     private static final String JSON = """
             {
                "title": "FullObjective", "ownerId": 1, "ownerFirstname": "Bob", "ownerLastname": "Kaufmann",
                "teamId": 1, "teamName": "Team1", "quarterId": 1, "quarterNumber": 3, "quarterYear": 2020,
                "description": "This is our description", "progress": 33.3
             }
             """;
+    private static final String DUPLICATE_OBJECTIVE = """
+            {
+                "objective": %s,
+                "keyResults": [%s,%s]
+            }
+            """.formatted(JSON, KEY_RESULT_METRIC_JSON, KEY_RESULT_ORDINAL_JSON);
```

Comment:
most likely tests depending on this will fail especially mockmvc tests, because the body is not necessarily correct JSON